src/bindings/python/requirements.txt

```diff
@@ -1,3 +1,3 @@
-numpy>=1.16.6,<2.1.0
+numpy>=1.16.6,<2.2.0
```

Comment:
@akuporos 
should we remove upper bound? Or at least <3.0

Reply:
We strongly recommend to not do this for numpy specifically.
They have history of minor releases that broke backward combability (and consequences for OV) and we should test it each time.

Reply:
historically, we did not have upper bound for numpy https://github.com/openvinotoolkit/openvino/blob/releases/2024/0/src/bindings/python/requirements.txt

what should we start to keep it after 2.x is introduced?

Reply:
yes, we had it for [LTS](https://github.com/openvinotoolkit/openvino/blob/releases/2023/3/src/bindings/python/requirements.txt) releases.
and the PR we are discussing will be a part of LTS as well.

from my point of view - at least the fact that numpy had 5 releases (incl bugfix releases) since June.
For me it means that the number of changes they made makes this branch a little bit unstable.

Reply:
I'd prefer to keep it like <3.0

Reply:
Not for LTS. 
For next releases - yes.